build_scripts/windows/scripts/build.cmd

```diff
@@ -134,6 +134,7 @@ if %errorlevel% neq 0 goto ERROR
 copy %REPO_ROOT%\build_scripts\windows\resources\CLI_LICENSE.rtf %BUILDING_DIR%
 copy %REPO_ROOT%\build_scripts\windows\resources\ThirdPartyNotices.txt %BUILDING_DIR%
 copy %REPO_ROOT%\NOTICE.txt %BUILDING_DIR%
+copy %REPO_ROOT%\build_scripts\windows\scripts\az.completion.ps1 %BUILDING_DIR%
```

Comment:
Should we put it to `%BUILDING_DIR%\Scripts\`?
It looks more formal but this folder is created by pip install.